docs/source/checks/tabular/integrity/plot_mixed_data_types.py

```diff
@@ -2,27 +2,78 @@
 """
 Mixed Data Types
 ****************
-"""
 
-#%%
+This notebooks provides an overview for using and understanding the mixed data types check.
 
-import pandas as pd
+**Structure:**
+
+* `What are Mixed Data Types? <#what-are-mixed-data-types>`__
+* `Run the Check <#run-the-check>`__
+* `Define a Condition <#define-a-condition>`__
+
+What are Mixed Data Types?
+==========================
+Mixed data types is a situation when a column contains numeric values as string type, or a column contains both numeric
+and string types. The check detects this situation whether most of the values are numeric (counting also numbers in
+string type as numeric) or most of the values are string.
+We want to be aware of this "types mix" since it may indicate a problem in the data collection pipeline, or represent a
+problem for the model's training.
```

Comment:
This doesn't describe clearly or accurately what the check does
(*also the API reference doesn't...)

Consider:
Shortening this (but also making it more accurate) and opening an issue to fix check, docstring, and notebook
(note that currently, the "condition logic" is implemented inside the check + no control on parameters)

Reply:
It might not be clear but what's not accurate in the explanation?

Reply:
> note that currently, the "condition logic" is implemented inside the check + no control on parameters
also not sure what you mean here, the condition there is regular